localization/react-intl/src/app/components/team/Newsletter/NewsletterComponent.json

```diff
@@ -59,6 +59,11 @@
     "description": "Title for newsletter settings page.",
     "defaultMessage": "Newsletter"
   },
+  {
+    "id": "newsletterComponent.helpContext",
+    "description": "Context description for the functionality of this page",
+    "defaultMessage": "Manage, draft, and schedule newsletters sent to all subscribers. <a href=\"{helpLink}\" target=\"_blank\" title=\"Learn more\">Learn more about newsletters</a>."
```

Comment:
I'm not sure how well these html messages will play out with translators, but we'll find out...

Reply:
interesting...well, let me know if they have any issues. Luckily since they are all added the same way, via the settings header component...if i need to change them, they will all be easy to find